### PR TITLE
Developlocal

### PR DESCRIPTION
**Pull request by danaharper151**
Pull request for buttons!! I got the buttons black, at the bottom, and the title at the top, center. Took forever, and I wanted to have some progress to show. I added the maps activity and connected it to the "Search" Button, but the app crashed when I click the button.  I will keep working on connecting the buttons to second pages, but this is what I have for now. Let me know if it will work, or if you need me to make changes to get it merge-able.
Dana

### CODE REVIEW

**Review by blargenfaarg — CHANGES_REQUESTED** [review]
Hi Dana, 
I took a look at the changes but I'm still seeing the default "Hello Android!" program in MainActivity.kt, looks like the changes weren't committed since I can't see any of the work you added to the program. 

Also, I believe we're supposed to be merging into the develop branch instead of main, hope this helps!

**Comment by danaharper151** [comment]
Hmm, I did pick develop, not main.. let me try again.
*Dana Harper*
***@***.***

***@***.***>

***@***.***>
